libs/api-client-tiger/src/gd-tiger-model/typeGuards.ts

```diff
@@ -26,3 +27,13 @@ export function isResultAttributeHeader(
 ): header is AttributeExecutionResultHeader {
     return (header as AttributeExecutionResultHeader).attributeHeader !== undefined;
 }
+
+export function isVisualizationObjectsItem(
+    visualizationObject: unknown,
+): visualizationObject is VisualizationObjectsItem {
+    return (visualizationObject as VisualizationObjectsItem).type === "visualizationObject";
+}
+
+export function isFilterContextData(filterContext: unknown): filterContext is FilterContextData {
+    return (filterContext as FilterContextData).data.type === "filterContext";
```

Comment:
This can fail if the object passed in does not have `data` defined.

Reply:
BE model change fix this - there is no longer `data` level